js/src/main/scala/JSVyxal.scala

```diff
@@ -10,33 +9,35 @@ import scala.scalajs.js.JSConverters.*
 @JSExportTopLevel("Vyxal")
 object JSVyxal:
   @JSExport
-  def execute(code: String, inputs: String, flags: String): String =
+  def execute(
+      code: String,
+      inputs: String,
+      flags: String,
+      printFunc: String
```

Comment:
Why not take a `js.Function1[String, Unit]`? That can easily be converted to a Scala function (`String => Unit`)

Reply:
This whole thing was painful enough that I didn't want to try experimenting with forcing it to scala types explicitly